app/javascript/stylesheets/workTypes.scss

```diff
@@ -0,0 +1,41 @@
+.work-types {
+  a {
+    text-decoration: none;
+
+    &:after {
+      content: '\f078';
+      font-family: Fontawesome;
+      padding-left: 0.5rem;
+    }
+
+    &.collapsed:after {
+      content: '\f054';
+    }
+  }
+
+  h6 {
+    color: $stanford-digital-red;
+    font-weight: bold;
+  }
```

Comment:
This is new, to bring attention to the unique requirement of music-type works. (Will also be used for mixed material-type works eventually.)